tasks/main.yml

```diff
@@ -16,6 +16,7 @@
     image: '{{ postgres_docker_image }}:{{ postgres_docker_image_tag }}'
     restart_policy: always
     state: started
+    memory: '{{ container_memory_limit }}'
```

Comment:
http://docs.ansible.com/ansible/playbooks_filters.html#omitting-undefined-variables-and-parameters